uno.config.ts

```diff
@@ -75,6 +75,15 @@ export default defineConfig({
       },
       scrim: 'hsl(var(--c-scrim))',
       shadow: 'hsl(var(--c-shadow))',
+      logo: {
+        i: 'hsl(var(--c-logo-i))',
+        np: {
+          dark: 'hsl(var(--c-logo-np-dark))',
+          light: 'hsl(var(--c-logo-np-light))',
+        },
```

Comment:
```suggestion
        np: 'hsl(var(--c-logo-np))',
```

Reply:
wait but there is 2 diff colours black and white

Reply:
Yes. 

`--c-logo-np` points to either the dark or light color depending on whether the `dark` class is present

Reply:
The dark variant will be set here

https://github.com/np-inprove/app/pull/18/files#diff-cd863d7dba00ecc23346c7d7a277404fb9f3400715c93f64f1988ac435b01fddR94